packages/browser/src/BacktraceBrowserRequestHandler.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    BacktraceAttachment,
+    BacktraceReportSubmissionResult,
+    BacktraceRequestHandler,
+    DEFAULT_TIMEOUT,
+} from '@backtrace/sdk-core';
+import { BacktraceData } from '@backtrace/sdk-core/src/model/data/BacktraceData';
+
+export class BacktraceBrowserRequestHandler implements BacktraceRequestHandler {
+    private readonly UPLOAD_FILE_NAME = 'upload_file';
+    private readonly _timeout: number;
+
+    constructor(
+        private readonly _options: {
+            url: string;
+            token?: string;
+            timeout?: number;
+        },
+    ) {
+        this._timeout = this._options.timeout ?? DEFAULT_TIMEOUT;
+    }
+    public async postError<T>(
+        submissionUrl: string,
+        data: BacktraceData,
+        attachments: BacktraceAttachment[],
+    ): Promise<BacktraceReportSubmissionResult<T>> {
+        const formData = this.createFormData(JSON.stringify(data), attachments);
+        return this.post(submissionUrl, formData);
+    }
+
+    public async post<T>(
+        submissionUrl: string,
+        payload: string | FormData,
+    ): Promise<BacktraceReportSubmissionResult<T>> {
+        const xmlHttpRequest = new XMLHttpRequest();
```

Comment:
Is there a reason why you're using XHR instead of fetch? All modern browsers seem to support it.

https://caniuse.com/fetch

Reply:
This is the copy paste code from Backtrace-js. We can use fetch